packages/react-core/src/components/TextInputGroup/examples/TextInputGroup.md

```diff
@@ -10,22 +10,34 @@ import TimesIcon from '@patternfly/react-icons/dist/esm/icons/times-icon';
 
 ## Examples
 
-### Basic
+### Basic text input group 
+
+A basic text input group updates its contents as users enter text.
 
 ```ts file="./TextInputGroupBasic.tsx"
 ```
 
-### Disabled
+### Disabled text input group
+
+To prevent users from making edits to a text input group, you can disable it using `isDisabled`.
+
+When you disable a text input group, add an `aria-label` to provide context to assistive technologies.
```

Comment:
Typically an `aria-label` will be needed for any TextInputGroup, disabled or not, if there's no `label` element tied to the `input` itself.

We could move this line to the basic example and tweak it a little, or we could maybe just remove it and have it live int he accessibility docs once added, wdyt?

Reply:
Ah okay, fair point! Let's have it in the accessibility docs then